### ISSUE TITLE
Pull Requests - priority list Q3

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#864]
Priority list for `whosonfirst-data` PRs, Q3.

**High priority:**
_Top list items more important than bottom items._

- [x] #852 - 📋 Updating wof:name, name:eng_x_preferred, and {lang}_x_preferred for non-locality placetypes
- [x] #859 - 📋 Update bad French, German, Chinese and Dutch language code names
- [x] #876 - ⏳  New locality features from Natural Earth
- [x] #765 - 🎈 New Plymouth, Monsterrat record locality new features
- [x] #786 - ⏳ New admin0 naturalearth props, alt_geoms, and empire geoms admin-0 country enhancement new features

**Lower priority:**
_Top list items more important than bottom items._

- [x] #857 - 📋 Fix Ireland country codes (not Iran!)
- [ ] #700 - ⏳ New and updated county features for Austria
- [x] #697 - 🎈 Updated geometries for Austrian region records, new alt-geoms
- [ ] #688 - 🎈 New geometries and alt_geom records for Australia
- [x] #763 - 🎈 Updates to dependency hierarchies
- [ ] #804 - 🎈 Updated neighbourhood records with dual hierarchies
- [x] #791 - 🎈 Updated campus records - Boston, San Diego, Oakland
- [x] #514 - 🎈 Updates to Finland region records
- [x] #619 - 🎈 Merged two Sweden region records of Jämtland
- [x] #607 - 🎈 Updated duplicate locality records in Finland
- [x] #560 - 🎈 Dissolved duplicate county records in Belgium, superseded to new features
- [ ] #780 - ⏳ Quebec City/Levis neighbourhood import
- [ ] #421 - ⏳ Vancouver (and surrounding) neighbourhood imports
- [x] #881 - ⏳ Buenos Aires updates

### Legend 

- 📋 property edits only
- 🎈 easy sailing (not many features)
- ⏳ moderate number of features
- ⏳⏳ large number of features
- 📝 PIP in progress (by Aaron)
- 📥 PIP in review (with Stephen)

**Comment by thisisaaronland** [comment]
Please don't include things that are "almost done". This list needs to be things that are ready to go.

**Comment by stepps00** [comment]
@thisisaaronland - all merge conflicts in linked PRs have been resolved. The above PRs are all ready to be processed and imported.

**Comment by thisisaaronland** [comment]
To date everything has been processed except #780 which I'd like to leave for @dphiffer to test with.

The following issues have been processed but are waiting on tweaks, etc. by @stepps00 : #700 #688 #804

**Comment by stepps00** [comment]
@thisisaaronland - I have added #421 to this list, under "lower priority".

I am taking a look at #700 #688 #804 and will update this issue once conflicts/tweaks are worked out.

**Comment by stepps00** [comment]
Notes:

- #688: It looks like [these updates](https://github.com/whosonfirst-data/whosonfirst-data/commit/0a01d7aed0944d9e25c50aba5000b2415bdb9409) in Australia were merged to `master`. It doesn't looks like anything else is needed for this issue.

**Comment by stepps00** [comment]
I've added #881 to this list, it is an external contribution to fix neighbourhoods in Buenos Aires (through a PR, not the typical zip file workflow).

**Comment by stepps00** [comment]
Closing and replacing with https://github.com/whosonfirst-data/whosonfirst-data/issues/1002.
